docs/architecture.md

```diff
@@ -1,62 +1,56 @@
+collection rather than centralized collection. If you centrally locate your syslog server, the UDP and (stateless)
+TCP traffic cannot adjust and data loss will occur.
 
 ## syslog Data Collection at Scale
 As a best practice, do not co-locate syslog-ng servers for horizontal scale and load balance to them with a front-side load balancer:
 
-* Attempts to load balance for scale can cause more data loss due to normal device operations
-and attendant buffer loss. A simple, robust single server (or shared-IP cluster) will provide better performance.
+* Attempting to load balance for scale can cause more data loss due to normal device operations
+and attendant buffer loss. A simple, robust single server or shared-IP cluster provides the best performance.
 
-* Front-side load balancing will cause inadequate data distribution on the upstream side, leading to data unevenness on the indexers.
+* Front-side load balancing causes inadequate data distribution on the upstream side, leading to data unevenness on the indexers.
 
 ## High availability considerations and challenges
 
-Load balancing for high availability does not work well for stateless, unacknowledged syslog traffic. More data will be lost than if you use a more simple design such as vMotioned VMs.  With syslog, the protocol itself is prone to loss, and there
-will be data loss (similar to CD-quality (lossless) vs. MP3).  Syslog data collection can be made, at best, "Mostly Available".
+Load balancing for high availability does not work well for stateless, unacknowledged syslog traffic. More data is preserved when you use a more simple design such as vMotioned VMs.  With syslog, the protocol itself is prone to loss, and Syslog data collection can be made, at best, "Mostly Available".
 
 ## UDP vs. TCP
 
 Run your syslog configuration on UDP rather than TCP.
 
-The syslogd daemon was originally configured to use UDP for log forwarding to reduce overhead, as UDP's streaming method does not require the overhead of establishing a network session. 
+The syslogd daemon optimally uses UDP for log forwarding to reduce overhead. This is because UDP's streaming method does not require the overhead of establishing a network session. 
 UDP also reduces network load on the network stream with no required receipt verification or window adjustment.
-Although TCP uses ACKS and there should not be data loss, loss cann occur when:
+
+TCP uses acknowledgements (ACKS) to avoid data loss, however, loss can still occur when:
 
 * The TCP session is closed: Events published while the system is creating a new session are lost. (Closed Window Case)
-* The remote side is busy and can not ACK fast enough: Events are lost due to a full local buffer.
-* A single ACK is lost by the network and the client closes the connection: Local and remote buffer are lost.
+* The remote side is busy and cannot acknowledge fast enough: Events are lost due to a full local buffer.
```

Comment:
Can we specify ACK somehow as acknowledgement signal

Reply:
i replaced "acknowledgement" with "acknowledgement signal". I can also change it back to ACKS if you feel that works better for the audience.